e2e/wdio/driver/wdio.ts

```diff
@@ -290,3 +297,12 @@ export function clickAndDragElement(locationX: number, locationY: number, newLoc
         ]
     }]);
 }
+
+
+export function saveElement(selector: string, tag: string, options?: object, index: number = 0): void {
```

Comment:
maybe this could be "saveElementImg" or "saveElementScreenshot" to make it clear what the function does?

Reply:
Done